fedimint-atomic-broadcast/src/session.rs

```diff
@@ -72,24 +72,28 @@ pub async fn run(
     let (signature_sender, signature_receiver) = watch::channel(None);
     let (terminator_sender, terminator_receiver) = futures::channel::oneshot::channel();
 
-    let aleph_handle = tokio::spawn(aleph_bft::run_session(
-        config,
-        aleph_bft::LocalIO::new(
-            DataProvider::new(keychain.clone(), item_receiver, signature_receiver),
-            FinalizationHandler::new(unit_data_sender),
-            backup_saver,
-            backup_loader,
+    let aleph_handle = spawn(
+        "aleph run session",
+        aleph_bft::run_session(
+            config,
+            aleph_bft::LocalIO::new(
+                DataProvider::new(keychain.clone(), item_receiver, signature_receiver),
+                FinalizationHandler::new(unit_data_sender),
+                backup_saver,
+                backup_loader,
+            ),
+            Network::new(network_data_receiver, outgoing_message_sender.clone()),
+            keychain.clone(),
+            Spawner::new(),
+            aleph_bft::Terminator::create_root(terminator_receiver, "Terminator"),
         ),
-        Network::new(network_data_receiver, outgoing_message_sender.clone()),
-        keychain.clone(),
-        Spawner::new(),
-        aleph_bft::Terminator::create_root(terminator_receiver, "Terminator"),
-    ));
+    )
+    .expect("some handle");
```

Comment:
The reason why there will always be a handle is that this only runs on non-WASM platforms. Maybe we want to mention that in the expect?